lib/matplotlib/cbook.py

```diff
@@ -784,14 +784,15 @@ def is_scalar_or_string(val):
     return is_string_like(val) or not iterable(val)
 
 
-def _string_to_bool(s):
+def string_to_bool(s):
+    """Parses the string argument as a boolean"""
```

Comment:
For this to be a general cbook function, I think it should accept case-insensitive "true" and "false" as well as "on" and "off".